### PR TITLE
fix: cookbooks review

### PR DESCRIPTION
**Pull request by sr-remsha**

### CODE REVIEW

File: dial-cookbook/examples/how_to_call_image_to_text_applications.ipynb [hunk 1]
```diff
@@ -6,11 +6,11 @@
    "source": [
     "# How to call image-to-text DIAL applications\n",
     "\n",
-    "This notebook covers how to call image-to-text applications via [DIAL API chat/completions](https://epam-rail.com/dial_api#/paths/~1openai~1deployments~1%7BDeployment%20Name%7D~1chat~1completions/post) call.\n",
+    "From [this notebook](https://github.com/epam/ai-dial/blob/main/dial-cookbook/examples/how_to_call_image_to_text_applications.ipynb), you will learn how to call image-to-text DIAL applications via [DIAL API chat/completions](https://epam-rail.com/dial_api#/paths/~1openai~1deployments~1%7BDeployment%20Name%7D~1chat~1completions/post) call.\n",
     "\n",
-    "**DIAL application** is a general term, which encompasses model adapters and application with custom logic.\n",
+    "**DIAL Application** is a general term, which encompasses model Adapters and Application with any custom logic.\n",
```
Comment (adubovik) [inline dial-cookbook/examples/how_to_call_image_to_text_applications.ipynb:11]: It's strange to have **A**pplication and **A**dapters capitalized. They are not proper names.

File: dial-cookbook/examples/how_to_call_image_to_text_applications.ipynb [hunk 2]
```diff
@@ -420,63 +405,47 @@
...
-    "output = llm.stream(input=[HumanMessage(content=\"Hello world!\")])\n",
-    "completion = \"\"\n",
-    "for chunk in output:\n",
-    "    print(chunk.dict())\n",
-    "    completion += chunk.content\n",
-    "print(f\"Completion: {completion!r}\")\n",
-    "assert completion == \"Image was generated successfully\", \"Unexpected completion\""
+    "\n",
+    "try:\n",
+    "    output = llm.stream(input=[HumanMessage(content=\"Hello world!\", additional_kwargs=extra_fields)])\n",
```
Comment (adubovik) [inline dial-cookbook/examples/how_to_call_image_to_text_applications.ipynb:442]: Typo. Here should be an empty content: `content=""` as in other examples in the same article.

File: dial-cookbook/examples/how_to_call_image_to_text_applications.ipynb (hunk 1 above)
Comment (sr-remsha) [reply dial-cookbook/examples/how_to_call_image_to_text_applications.ipynb]: fixed

File: dial-cookbook/examples/how_to_call_image_to_text_applications.ipynb [hunk 3]
```diff
@@ -89,12 +87,12 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Helpers to display images in the notebook:"
+    "**Step 4**: helpers to read images from disk and display images in the notebook:"
```
[the remark this answers is not in the document]
Comment (sr-remsha) [reply dial-cookbook/examples/how_to_call_image_to_text_applications.ipynb]: fixed

File: sidebars.js [hunk 4]
```diff
@@ -57,7 +57,7 @@ const sidebars = {
     },
     {
       type: 'category',
-      label: 'Cookbook',
+      label: 'Cookbooks',
```
[the remark this answers is not in the document]
Comment (sr-remsha) [reply sidebars.js]: fixed

File: dial-cookbook/examples/how_to_call_image_to_text_applications.ipynb (hunk 2 above)
Comment (sr-remsha) [reply dial-cookbook/examples/how_to_call_image_to_text_applications.ipynb]: fixed

**Review by adubovik — APPROVED** [review]

**Pull request merged**